src/test/scala/com/redislabs/provider/redis/df/DataframeBenchmarkSuite.scala

```diff
@@ -0,0 +1,114 @@
+package com.redislabs.provider.redis.df
```

Comment:
Would it make sense to move all df benchmark traits to `/df/benchmark` package and implementations to `/df/benchmark/cluster` ?

Reply:
sure